command/kv_helpers.go

```diff
@@ -110,8 +110,24 @@ func addPrefixToVKVPath(p, mountPath, apiPrefix string) string {
 	case p == mountPath, p == strings.TrimSuffix(mountPath, "/"):
 		return path.Join(mountPath, apiPrefix)
 	default:
-		p = strings.TrimPrefix(p, mountPath)
-		return path.Join(mountPath, apiPrefix, p)
+		tp := strings.TrimPrefix(p, mountPath)
+		for {
```

Comment:
I know that this was already `switch` statement but it might be easier to read if it were refactored to be a single if where we check for the following with the same early return: 

```
if p == mountPath || p == strings.TrimSuffix(mountPath, "/")
```

the following loop using the trimmed prefix can then follow at the same indentation level as the `if` statement.